code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1230,6 +1230,11 @@
 			to_chat(affected_mob, span_warning("A phantom limb hurts!"))
 			affected_mob.say("Why are we still here, just to suffer?", forced = /datum/reagent/toxin/bonehurtingjuice)
 
+/datum/reagent/milk/used_on_fish(obj/item/fish/fish)
+	if(HAS_TRAIT(fish, TRAIT_FISH_MADE_OF_BONE))
+		fish.adjust_health(fish.health - 30)
+		return TRUE
```

Comment:
Looks like a duplicate of `code/modules/reagents/chemistry/reagents/drinks/drink_reagents.dm` line 229, pick one

Reply:
Ah, this was meant for the bone hurting juice. Good effing catch!